src/CacheTagsGetterTrait.php

```
@@ -57,9 +58,16 @@ protected function getCacheTags($contentfulResult): array
             }
 
             foreach ($fields as $field) {
+                $childTags = null;
+                try {
+                    $childTags = $this->getCacheTags($contentfulResult->{'get' . ucfirst($field->getId())}());
+                } catch (Exception $e) {
+                    // fail silently to avoid problems with corrupt entries
```

Fail silently without any log?

---

Added logging